pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -160,6 +160,11 @@ let
       ./patches/no-build-timestamps.patch
       # For bundling Widevine (DRM), might be replaceable via bundle_widevine_cdm=true in gnFlags:
       ./patches/widevine-79.patch
+      # Fix WebRTP affecting screen sharing. These fixes were integrated into chromium 105
+      # See https://webrtc-review.googlesource.com/c/src/+/267002
+      ./patches/chromium-webrtc-fix.patch
```

Comment:
Why are we not fetchpatch'ing those?

Reply:
The patch is from the WebRTC sub-project. To apply it in the 'chrome' tree I had to change the path from 'a/modules/...' to 'a/third_party/webrtc/modules/...'. I don't know how to automate this.

Reply:
We have extraPrefix for that https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/fetchpatch/default.nix#L14